scanner/src/lib/parse.ts

```diff
@@ -10,31 +10,33 @@ if (typeof window !== "undefined") {
   }
 }
 
-const PRESET_JWKS = {
-  keys: [
-    {
-      alg: "RS256",
-      kid: "01",
-      kty: "RSA",
-      n:
-        "yicAxwnv46tR82buQHN2b2dDLOmZ0iJYR6GPwNOShlCdN-N5N6spvAbe3UNb6ymN8AtW5o9_6Rm6gpungvlnDtjyZfPnNzAryjR8Z1" +
-        "Jvb33spr7zcLWG8TzL1l5r5b2zMnWXaDOZffAxa3SWZ14ZufAAA97fW3oBJXYr4H0zB8aFNuPDtYw4puN4s3M1vQH6OPgXDkSNYBv5Zbx" +
-        "lKasPXdFf_fXA9u4p9JMmtNS3fL7RibMqbrf4jCkATwVqA_Ui9vlgudZeHqW0tpzuXT3FVyVN-eFQwq6Hk3vY8iqRD9Skbpol9vHQLKY8" +
-        "px5tZZTxVtNcLhTKzNToMWQMykuq0Q",
-      e: "AQAB",
-    },
-  ],
-}
+// const PRESET_JWKS = {
+//   keys: [
+//     {
+//       alg: "RS256",
+//       kid: "01",
+//       kty: "RSA",
+//       n:
+//         "yicAxwnv46tR82buQHN2b2dDLOmZ0iJYR6GPwNOShlCdN-N5N6spvAbe3UNb6ymN8AtW5o9_6Rm6gpungvlnDtjyZfPnNzAryjR8Z1" +
+//         "Jvb33spr7zcLWG8TzL1l5r5b2zMnWXaDOZffAxa3SWZ14ZufAAA97fW3oBJXYr4H0zB8aFNuPDtYw4puN4s3M1vQH6OPgXDkSNYBv5Zbx" +
+//         "lKasPXdFf_fXA9u4p9JMmtNS3fL7RibMqbrf4jCkATwVqA_Ui9vlgudZeHqW0tpzuXT3FVyVN-eFQwq6Hk3vY8iqRD9Skbpol9vHQLKY8" +
+//         "px5tZZTxVtNcLhTKzNToMWQMykuq0Q",
+//       e: "AQAB",
+//     },
+//   ],
+// }
```

Comment:
Delete, don't comment out. Version control means you can always go back in history to look up things if you really need them, and I don't think we will need this anymore as it was for COSE signing which we will not be verifying on the client anymore.

```suggestion
```